pkgs/by-name/ac/ace-of-penguins/package.nix

```diff
@@ -10,18 +10,21 @@
   zlib,
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "ace-of-penguins";
   version = "1.4";
 
   src = fetchurl {
-    url = "http://www.delorie.com/store/ace/ace-${version}.tar.gz";
+    url = "http://www.delorie.com/store/ace/ace-${finalAttrs.version}.tar.gz";
     hash = "sha256-H+47BTOSGkKHPAYj8z2HOgZ7HuxY8scMAUSRRueaTM4=";
   };
 
   patches = [
     # Fixes a bunch of miscompilations in modern environments
     ./fixup-miscompilations.patch
+    # make-imglib.c:205:5: error: 'return' with no value, in function returning non-void [-Wreturn-mismatch]
+    # imagelib.c:109:17: error: implicit declaration of function 'malloc' [-Wimplicit-function-declaration]
+    ./fix-gcc-14.patch
```

Comment:
has this been up streamed ?

Reply:
This is very old software, so no.
But perhaps other repos have a fetcpatch-able patch to use.

Reply:
it's ok as it is, just checking